tiatoolbox/utils/misc.py

```diff
@@ -338,6 +337,77 @@ def contrast_enhancer(img, low_p=2, high_p=98):
     return np.uint8(img_out)
 
 
+def read_point_annotations(input_table):
+    """Read annotations as pandas DataFrame.
+
+    Args:
+        input_table (str or pathlib.Path or ndarray or pd.DataFrame): path to csv, npy
+         or json or an ndarray. first column in the table represents x position, second
+         column represents y position. The third column represents the class. If the
+         table has headers, the header should be x, y & class.
+
+    Returns:
+        pd.DataFrame: DataFrame with x, y location and class type.
+
+    Examples:
+        >>> from tiatoolbox.utils.misc import read_point_annotations
+        >>> labels = read_point_annotations('./annotations.csv')
+
+    """
+    if isinstance(input_table, (str, pathlib.Path)):
+        _, _, suffix = split_path_name_ext(input_table)
+
+        if suffix == ".npy":
+            out_table = np.load(input_table)
+            if out_table.shape[1] == 2:
+                out_table = pd.DataFrame(out_table, columns=["x", "y"])
+                out_table["class"] = None
+            else:
+                out_table = pd.DataFrame(out_table, columns=["x", "y", "class"])
+
+        elif suffix == ".csv":
+            out_table = pd.read_csv(input_table, sep=None, engine="python")
+            if "x" not in out_table.columns:
+                out_table = pd.read_csv(
+                    input_table,
+                    header=None,
+                    names=["x", "y", "class"],
+                    sep=None,
+                    engine="python",
+                )
+            if out_table.shape[1] == 2:
+                out_table["class"] = None
+
+        elif suffix == ".json":
```

Comment:
Same as above, need to wrap within `try` block. One more thing, I'm not sure what should be the format within `json` here, do you want it as follow or sthg else?
```
{
'x' : [0,1,2,3],
'y' : [1,2,3,4],
'class' : [0, 0, 1, 1]
}
```

Reply:
It will be as you mentioned above. I generated json using pd.to_json so I am assuming this to be default format.

Reply:
Resolved in this commit as discussed https://github.com/TIA-Lab/tiatoolbox/commit/854f4549144f618309dd275f1026120f41f41888